domains/authorization/Proxy.IntegrationTests/V1EndpointTests.cs

```diff
@@ -0,0 +1,225 @@
+using System.Collections;
+using System.Net;
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
+using System.Text;
+using System.Text.Json;
+using FluentAssertions;
+using Proxy.Controllers;
+using Proxy.IntegrationTests.Setup;
+using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
+
+namespace Proxy.IntegrationTests;
+
+public class V1EndpointTests(ProxyIntegrationTestFixture fixture) : IClassFixture<ProxyIntegrationTestFixture>
+{
+    private HttpClient CreateClientWithOrgIds(List<string> orgIds) => fixture.Factory.CreateAuthenticatedClient(orgIds: orgIds);
+    private HttpClient CreateClientWithOrgAsSub(string sub) => fixture.Factory.CreateAuthenticatedClient(sub: sub);
+
+
+    // [Fact]
+    // public async Task GivenB2C_WhenV20250101GetEndpointsAreUsed_WithInvalidOrgnaisationId_Return403Forbidden()
```

Comment:
Why was this commented out. This seems like a pretty important case to have asserted, no? 🤔If the proxy does not return 403 forbidden, then it is not doing its job

Reply:
It's an important test. But currently this is failing with 500 because of IdentityDescriptor. So this needs to be fixed in other issue. Left test here so we can bring it back when we have solution for it. It's the stuff we talked about :)

Reply:
Added comment so people can read why it's commented out.